core/src/Streamly/Internal/Unicode/Char/Parser.hs

```diff
@@ -77,7 +76,7 @@ import qualified Streamly.Internal.Data.Parser as Parser
 
 -- XXX It may be possible to implement faster predicates for ASCII byte stream.
 -- We can measure if there is a signficant difference and if so we can add such
--- predicates to Streamly.Unicode.Char.Parser.Latin1.
+-- predicates to Streamly.Unicode.Parser.Latin1.
```

Comment:
What is `Unicode.Parser.Latin1`? Where should `Latin1` stuff go in? @harendra-kumar